MediaBrowser.Model/Drawing/ImageOrientation.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace MediaBrowser.Model.Drawing
 {
+#pragma warning disable CA1008 // Disabling Null default rule check since the enum is already shipped and default is handled in GetSKEncodedOrigin()
```

Comment:
Just put this at the top of the file and remove the restore

Reply:
Done.